src/componentSetBuilder.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as path from 'path';
 import { ComponentSet, RegistryAccess } from '@salesforce/source-deploy-retrieve';
-import { ComponentLike } from '@salesforce/source-deploy-retrieve/lib/src/resolve/types';
+import { ComponentLike } from '@salesforce/source-deploy-retrieve';
```

Comment:
line 9 and 10 should be combined, this should be applied to every duplicate import